source/fuzz/protobufs/spvtoolsfuzz.proto

```diff
@@ -742,6 +743,88 @@ message TransformationAddLoopPreheader {
 
 }
 
+message TransformationAddLoopToCreateIntConstantSynonym {
+  // A transformation that uses a loop to create a synonym for an integer
+  // constant C (scalar or vector) using an initial value I, a step value S and
+  // a number of iterations N such that C = I - N * S. For each iteration, S is
+  // added to the total.
+  // The loop can be made up of one or two blocks, and it is inserted before a
+  // loop with a single predecessor. In the one-block case, it is of the form:
+  //
+  //            %loop_id = OpLabel
+  //             %ctr_id = OpPhi %int %int_0 %pred %incremented_ctr_id %loop_id
+  //            %temp_id = OpPhi %intI %I %pred %eventual_syn_id %loop_id
+  //    %eventual_syn_id = OpISub %intI %temp_id %step_val_id
+  // %incremented_ctr_id = OpIAdd %int %ctr_id %int_1
+  //            %cond_id = OpSLessThan %bool %incremented_ctr_id %num_iterations_id
+  //                       OpLoopMerge %block_after_loop_id %loop_id %none
+  //                       OpBranchConditional %cond_id %loop_id %block_after_loop_id
+  //
+  // Which can be translated, assuming that N > 0, to:
+  // for (int ctr = 0; ctr < N; ctr++) S = S - I
```

Comment:
I think that you meant: 'I = I - S'

Reply:
The comment is indeed wrong, thanks for spotting this. I changed it and used an additional variable syn, to make it clear that the value of I does not change:

```
  // int syn = I;
  // for (int ctr = 0; ctr < N; ctr++) syn = syn - S;
```